ci/test-stable.sh

```diff
@@ -28,10 +28,10 @@ for program in programs/native/*; do
 done
 
 # Run integration tests serially
-for test in tests/*.rs; do
+for test in tests/*.rs wallet/tests/*.rs; do
   test=${test##*/} # basename x
   test=${test%.rs} # basename x .rs
-  _ cargo test --verbose --features="$FEATURES" --test="$test" -- --test-threads=1 --nocapture
+  _ cargo test --all --verbose --features="$FEATURES" --test="$test" -- --test-threads=1 --nocapture
```

Comment:
Yes. Cargo complains it can’t find whatever named test in submodule/tests without it.

Reply:
@mvines, what do you think? At a glance, this looks like it could become painfully slow. Is adding `--all` here the right thing to do or should it be a `cd wallet && `?

Reply:
Incidentally, my original plan was to make the match generic,`for test in tests/*.rs */tests/*.rs;`, so as to not require any upkeep for future submodule integration tests. But the tests in `ledger-tool/tests` need some attention for that to work.